ossdbtoolsservice/metadata/metadata_service.py

```diff
@@ -91,6 +112,13 @@ def _list_metadata(self, owner_uri: str) -> List[ObjectMetadata]:
                 metadata_list.append(ObjectMetadata(None, object_type, None, object_name, schema_name))
         return metadata_list
 
+    def _schema_metadata(self, owner_uri: str) -> str:
+        # Get current connection
+        connection_service = self._service_provider[constants.CONNECTION_SERVICE_NAME]
+        connection: ServerConnection = connection_service.get_connection(owner_uri, ConnectionType.DEFAULT)
+        schema = SchemaMetadata(connection.cursor(), 'public')
```

Comment:
Curious why do we only support 'public' schema? In the SchemaMetadata, init set param schema's default value as public

Reply:
Good point, we're using the public schema for now, but I'll let the default argument take care of that rather than specifying it again here.